### ISSUE TITLE
Research implementing a11y scans for itdashboard

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#545]
Specifcally:
*  how many HTML tags have missing attribute text (e.g., missing/empty links and buttons, missing form input labels, missing document language
* how many color contrast issues were found on a website homepage out of compliance with the Section 508 standards set by the U.S. Access Board, which requires agencies to conform to WCAG 2.0’s Level A and Level AA Success Criteria and Conformance Requirements.
* how many website images and other non-text content, including user interface elements, are missing a text description.
